apps/keira/src/assets/i18n/en.json

```diff
@@ -85,7 +85,8 @@
       "SKINNING_LOOT": "Skinning Loot",
       "SPAWN": "Spawn",
       "SPAWN_ADDON": "Spawn Addon",
-      "SMARTAI": "SmartAI"
+      "SMARTAI": "SmartAI",
+      "TEXT": "Text"
```

Comment:
we should add this new key in all other languages, it's okay to put the text in English as temporary and it can be translated later (but they key should exist)